src/_locales/en/messages.json

```diff
@@ -75,31 +75,31 @@
     "message": "Miscellaneous"
   },
   "passff_prefs_mark_fillable": {
-    "message": "Mark fillable input fields with icon"
+    "message": "Mark fillable forms with an icon-shaped form menu"
   },
   "passff_prefs_submit_fillable": {
-    "message": "Submit when selected via icon menu"
+    "message": "Submit when selected via the form menu"
   },
   "passff_prefs_handle_http_auth": {
     "message": "Handle HTTP authentication"
   },
   "passff_prefs_directories_first": {
-    "message": "List directories first in password repository"
+    "message": "List directories first in the toolbar menu"
   },
   "passff_prefs_case_insensitive_search": {
     "message": "Case-insensitive searching"
   },
   "passff_prefs_enter_behavior": {
-    "message": "Behavior of enter key"
+    "message": "Behavior of Enter key"
   },
   "passff_prefs_fields_description": {
-    "message": "Fields are used to find data inside your passwords repository. PassFF will try to find the login, password or url value inside the password data (using the standard syntax) or trying to find a matching child node name"
+    "message": "Fields are used to find data inside your password repository. PassFF looks login, password and URL up in the password file. If it's a directory, PassFF looks them up in the files of this directory"
   },
   "passff_prefs_password_input_names_label": {
-    "message": "Password input names"
+    "message": "Password form names"
```

Comment:
What we are refering to here, is not a form, it's only an input field in a form.

Reply:
Having fields for different entities is tough. Maybe there is a compromise: a [`form input`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input). But it complicates everything (2 words VS 1).

#### However, I have thought about a UX improvement that address this ambiguity

Put the *password/login input names* inputs in their own section called *Inputs* (or Form Inputs), like the Fields. And like the fields, add a text description such as:
`Input names are used by PassFF to find the fillable fields/forms/inputs in web pages.`
It separates the inputs names from the *Autofill* section, which is desirable since inputs names are also used to mark the inputs!

Preferences *Mark fillable inputs* and *Submit when selected via icon menu* could also be moved in this new section.

It might be the subject of a new PR.